diff --git a/src/app/soapy_connector.cpp b/src/app/soapy_connector.cpp
--- a/src/app/soapy_connector.cpp
+++ b/src/app/soapy_connector.cpp
@@ -3,8 +3,10 @@
 #include <iostream>
 #include <algorithm>
 #include <getopt.h>
 #include <vector>
+#include <SoapySDR/Modules.hpp>
+#include <SoapySDR/Registry.hpp>
 
 int main (int argc, char** argv) {
     Connector* connector = new SoapyConnector();
@@ -17,8 +19,9 @@ std::stringstream SoapyConnector::get_usage_string() {
     std::stringstream s = Connector::get_usage_string();
     s <<
         " -a, --antenna           select antenna input\n" <<
-        " -t, --settings          set sdr specific settings\n";
+        " -t, --settings          set sdr specific settings\n" <<
+        " -l, --listdrivers       list installed SoapySDR drivers\n";
     return s;
 }
 
@@ -26,7 +29,8 @@ std::vector<struct option> SoapyConnector::getopt_long_options() {
     std::vector<struct option> long_options = Connector::getopt_long_options();
     long_options.push_back({"antenna", required_argument, NULL, 'a'});
     long_options.push_back({"settings", required_argument, NULL, 't'});
+    long_options.push_back({"listdrivers", no_argument, NULL, 'l'});
     return long_options;
 }
 
@@ -37,12 +41,23 @@ int SoapyConnector::receive_option(int c, char* optarg) {
         case 't':
             settings = std::string(optarg);
             break;
+        case 'l':
+            listDrivers();
+            return 1;
         default:
             return Connector::receive_option(c, optarg);
     }
     return 0;
 }
 
+void SoapyConnector::listDrivers() {
+    for (const auto &mod : SoapySDR::listModules()) {
+        SoapySDR::loadModule(mod);
+    }
+    for (const auto &it : SoapySDR::Registry::listFindFunctions()) {
+        std::cout << it.first << std::endl;
+    }
+}
+
 void SoapyConnector::print_version() {
     std::cout << "soapy_connector version " << VERSION << std::endl;
     Connector::print_version();
diff --git a/src/app/soapy_connector.hpp b/src/app/soapy_connector.hpp
--- a/src/app/soapy_connector.hpp
+++ b/src/app/soapy_connector.hpp
@@ -35,5 +35,6 @@ class SoapyConnector: public Connector {
         int receive_option(int c, char* optarg) override;
         int setAntenna(std::string antenna);
         int setSettings(std::string settings);
+        static void listDrivers();
 };
