diff --git a/conf/CMakeLists.txt b/conf/CMakeLists.txt
index 9f45e15..0000000 100644
--- a/conf/CMakeLists.txt
+++ b/conf/CMakeLists.txt
@@ -29,7 +29,8 @@ file(WRITE "${CMAKE_BINARY_DIR}/direwolf.conf" "${file_content}")
 # says that we should use the /usr/lib/udev/rules.d location when building a package.
 # TODO:  I think the proper solution is to select the location based on whether
 # the application installation location is /usr/local or /usr.
-if(LINUX)
+option(INSTALL_UDEV_RULES "Install Direwolf udev rules" ON)
+if(LINUX AND INSTALL_UDEV_RULES)
   if (CMAKE_INSTALL_PREFIX STREQUAL "/usr/local") 
     install(FILES "${CUSTOM_CONF_DIR}/99-direwolf-cmedia.rules" DESTINATION /etc/udev/rules.d/)
   else()
