diff --git a/src/modules.cpp b/src/modules.cpp
index 44579c1..0000000 100644
--- a/src/modules.cpp
+++ b/src/modules.cpp
@@ -20,10 +20,15 @@
 
 static PyModuleDef pycsdrmodule = {
     PyModuleDef_HEAD_INIT,
-    .m_name = "digiham.modules",
-    .m_doc = "Python bindings for the digiham library",
-    .m_size = -1,
+    "digiham.modules",
+    "Python bindings for the digiham library",
+    -1,
+    nullptr,
+    nullptr,
+    nullptr,
+    nullptr,
+    nullptr,
 };
 
 PyMODINIT_FUNC
